### ISSUE TITLE
It seems that GitUp doesn't use ssh config file properly for multiple-host-key setup.

### ISSUE THREAD

**Issue by ShenghaiWang** [git-up/GitUp#676]
I have setup `~/.ssh/config` for different hosts to use different keys. The setup is working in terminal using git commands. However, when I use menu `Remotes/Fetch All Branches` in GitUp, it only works for one host and shows `Failed connecting to "origin" remote: Forbidden` for other hosts.

**Comment by MagFer** [comment]
**Issue**
It looks like GitUp isn't using the defined pair of keys of shh config file, find some related issues I've seen [#576](https://github.com/git-up/GitUp/issues/576
) [#35](https://github.com/git-up/GitUp/issues/35). The solution seems to be to have to add the keys to the ssh-agent every time you log in `ssh-add ~/.ssh/id_rsa`.

**Solution**
In order to automatice this process I've created an automation which is called every time the user logs in MacOS.

<img width="692" alt="image" src="https://user-images.githubusercontent.com/9061996/81052180-e8433000-8eba-11ea-9e5a-95ac25c3bff0.png">

<img width="466" alt="image" src="https://user-images.githubusercontent.com/9061996/81052191-ec6f4d80-8eba-11ea-89c0-b98e7e36fcc0.png">

https://developer.apple.com/library/archive/documentation/LanguagesUtilities/Conceptual/MacAutomationScriptingGuide/MakeaSystem-WideService.html

**Observations**
If doesn't work the first time, try to log out and log in again in your user session.

**Comment by ShenghaiWang** [comment]
Thanks for the suggestion. Yes, it works in my terminal after I manually add two keys. I am wondering why my GitUp cannot pick up this.

Did your GitUp work properly after you add these two keys?

**Comment by MagFer** [comment]
It didn't till I restarted the computer. Perhaps there's something that isn't loaded from GitUp after the session has started but I doesn't make sense. Give a try embedding the bash calls in an automator service called on session start, and restart the computer 😅🤞🏼

**Comment by ShenghaiWang** [comment]
Are you sure the second key of this setup works for you? On my laptop, only the first key added in the script works and I can use it to access one repo in GitUp, but I cannot access another repo which requires the second key. I even tried to change the order of the keys in the script, same problem.

**Comment by MagFer** [comment]
You are right @ShenghaiWang, despite load all keys (can be checked by calling `ssh-add -L`) both from GitUp and Terminal it's only posible to access the repo with the first key loaded in the bash script. Could this be related to be using bash rather than zsh, since is the native shell now in macOS Catalina?

<img width="535" alt="image" src="https://user-images.githubusercontent.com/9061996/81055581-e2504d80-8ec0-11ea-98c3-49654b5ad1b5.png">

**Comment by ShenghaiWang** [comment]
I was using `zsh`. Actually both keys are always working in my terminal on my laptop and that's why I thought it's an issue of GitUp.
